tasks/main.yml

```diff
@@ -36,21 +36,38 @@
   when: ansible_os_family == "Debian"
   tags: acmetool-install
 
-- name: Add RedHat/CentOS repo file
+- name: Add RedHat/CentOS/Rocky 7 repo file
```

Comment:
Do we need the 7 here?

Reply:
@Dhwaniartefact pointed me that this is correct because they are two different tasks. Sorry!